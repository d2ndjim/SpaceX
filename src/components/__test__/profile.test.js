import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../../redux/configureStore';
import Profile from '../profile/profile';

describe('Profiles test', () => {
  it('renders profiles', () => {
    const app = renderer
      .create(
        <Provider store={store}>
          <Profile />
        </Provider>,
      )
      .toJSON();
    expect(app).toMatchSnapshot();
  });
});
