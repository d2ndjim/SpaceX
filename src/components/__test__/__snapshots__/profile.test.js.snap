// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profiles test renders profiles 1`] = `
<section
  className="grid"
>
  <div
    className="mission"
  >
    <h2
      className="header"
    >
      My Missions
    </h2>
    <p>
      No Mission Joined!
    </p>
  </div>
  <div
    className="rocket"
  >
    <h2
      className="header"
    >
      My Rockets
    </h2>
    <p>
      No Rocket Reserved!
    </p>
  </div>
  <div
    className="dragon"
  >
    <h2
      className="header"
    >
      My Dragons
    </h2>
    <p>
      No Dragon Reserved!
    </p>
  </div>
</section>
`;
